services/provider/proto/provider.proto

```diff
@@ -169,6 +169,7 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
-    string desiredClientOperatorChannel = 1;
+    // Contains hash of desired client state
+    string desiredClientState = 1;
```

Comment:
`desiredClientState` is kind of misleading, as I would expect that this will derive from the state itself. 
Let me suggest a different name here: `DesiredConfigHash`.
The hash part in the name is important because it is not obvious that the value does not contain the state/data/config

Also, why is the reporter request designated a client/consumer ID? If not, how do we know which client this configuration belongs to?